release/src/router/rom/webs_scripts/merlin_webs_update.sh

```diff
@@ -93,11 +93,11 @@ if [ "$webs_state_flag" -eq "1" ]; then
 	if [ "$forsq" == "1" ]; then
 		echo "---- download SQ release note $fwsite/test/$releasenote_file0_US ----" >> /tmp/webs_upgrade.log
 		/usr/sbin/wget $wget_options $fwsite/test/$releasenote_file0_US -O $releasenote_path0
-		echo "---- $fwsite/test/$releasenote_file0 ----" >> /tmp/webs_upgrade.log
+		echo "---- $fwsite/test/$releasenote_file0_US ----" >> /tmp/webs_upgrade.log
```

Comment:
Note that $releasenote_file0 is not a valid variable
It should be $releasenote_file0_US

Reply:
SQ selease is unused in production, it's old QA/testing code that allows us developers to temporarily point to a different manifest.

Reply:
@RMerl 

I see, In such a case, I would normally comment out the code
It's just not the standard I'm accustomed to, since the variable will still be read and considered invalid whether it's used or not.

For debugging purposes, usually we comment out code.

Reply:
@RMerl 

Resolving as your commit: https://github.com/RMerl/asuswrt-merlin.ng/commit/90a9cc3a53e0415056949e0eec6318bb28244c9a deals with it in a cleaner manner than just patching the variable.